tier2/{{cookiecutter.project_slug}}/.github/workflows/contributors.yml

```diff
@@ -0,0 +1,55 @@
+on:
+    push:
+        branches:
+            - main
+    workflow_dispatch:
+
+jobs:
+    update-contributors:
+        runs-on: ubuntu-latest
+        name: Update contributors info in MAINTAINERS.md
+        permissions:
+            contents: write
+            pull-requests: write
+        steps:
+            # Update contributor list
+            - name: Contribute List
+              uses: akhilmhdh/contributors-readme-action@v2.3.10
+              env:
+                  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              with:
+                  # https://github.com/marketplace/actions/contribute-list#optional-parameters
+                  readme_path: MAINTAINERS.md
+                  use_username: false
+                  commit_message: "BOT: Update contributors info in MAINTAINERS.md"
+
+            # Update contributor count
+            - name: Checkout repository
+              uses: actions/checkout@v4
+
+            - name: Pull changes from contributors-readme-action
+              run: |
+                  git pull
```

Comment:
Unless the action that you used earlier doesn't make changes to the local branch you shouldn't have to pull changes since they should already be local.

Reply:
The previous Github action step pushes changes to the MAINTAINERS.md file, so the script needs to pull again or else the script complains about upstream changes and won't run